src/client/lazy-app/Modal/style.css

```diff
@@ -0,0 +1,171 @@
+dialog {
+  padding: 0;
+  width: 70vw;
+  max-width: 60ch;
+  max-height: 70vh;
+  cursor: auto;
+
+  box-shadow: 0 0 20px 0 rgba(0, 0, 0, 20%);
+
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+
+  font-size: 1.5rem;
+  color: var(--light-gray);
+
+  border: none;
+  border-radius: 10px;
+  overflow: hidden;
+
+  opacity: 0;
+  background-color: var(--off-black);
+
+  /* Note: dialog and ::backdrop are animated using the Animations API in the component file */
+
+  &::backdrop {
+    background-color: rgba(0, 0, 0, 30%);
+  }
+}
+
+@media (max-width: 720px) {
+  dialog {
+    width: 90%;
+    max-width: none;
+    font-size: 1.25rem;
+  }
+}
+
+@media (max-width: 480px) {
+  dialog {
+    font-size: 1rem;
+  }
+}
+
+.header,
+.footer {
+  max-height: 100%;
+  padding: 1em;
+
+  background-color: var(--dark-gray);
+}
+
+.header {
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+  align-items: center;
+  gap: 0.5em;
+  border-bottom: 1px solid rgba(0, 0, 0, 8%);
+  align-content: stretch;
+
+  /* Apply font size only to children so that padding is unaffected */
+  & > * {
+    font-size: 1.5em;
+  }
+}
+
+.footer {
+  border-top: 1px solid rgba(0, 0, 0, 8%);
+}
+
+.modal-title {
+  text-align: center;
+
+  /* Overflowing text gets ellipse-d */
+  overflow: hidden;
+  text-overflow: ellipsis;
+  width: 100%;
+  white-space: nowrap;
+}
+
+.modal-icon {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  opacity: 0.5;
+
+  svg {
+    font-size: 1em;
+    height: 1em;
+    width: 1em;
+  }
+}
+
+.close-button {
+  composes: unbutton from global;
+  flex: 0 0 auto;
+  height: 1em;
+  width: 1em;
+  background-color: var(--pink);
+  border-radius: 5px;
+  display: grid;
+  outline-color: var(
+    --medium-light-gray
+  ) !important; /* Overwrite default black outline */
```

Comment:
Where's this default coming from? As in, why does it need `!important`?

Reply:
The global `unbutton` sets the outline-color with the focus selector, so that overrides the default

...and now that I think of it, I should've just done the same and set it on focus - eliminating `!important`. Will do.